### PR TITLE
[Web Console] fix deprecated keyboard event method "keyCode" with "key"

### LINKED ISSUE #11946: Web Console making use of Deprecated KeyboardEvent "keyCode" (TS)
Example git line https://github.com/apache/druid/blob/29710789a438bd2c16bf2d1e556b96f8ccb57611/web-console/src/entry.ts#L106

### Affected Version
Latest
The Druid version where the problem was encountered.

### Description
TypeScript error on `KeyboardEvent.keyCode` method [Deprecation](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) . It should be replaced with the new and correct method as per the web standards.

I believe `KeyboardEvent.key` [method](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) should be used instead. 

Please include as much detailed information about the problem as possible.
```
1. typescript: 'keyCode' is deprecated. [6385]
```

### PR DESCRIPTION
**Pull request by AshishKapoor**
Fixes #11946.

### Description
Replace Keyboard event `keyCode` method with `key`. These changes increase code readability and get rid of deprecated method keyCode, [see](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode).

Should fix the TypeScript error respectively.

<hr>

##### Key changed/added classes in this PR
web-console/src/bootstrap/react-table-custom-pagination.tsx
web-console/src/entry.ts
web-console/src/views/query-view/query-output/column-rename-input/column-rename-input.tsx

<hr>

This PR has:
- [x] been self-reviewed.
   - [x] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/dev/license.md)
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold for [code coverage](https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md) is met.
- [ ] added integration tests.
- [x] been tested in a test Druid cluster.

### CODE REVIEW

**Comment by AshishKapoor** [comment]
Hey, @vogievetsky @suneet-s please review this small PR.

Thanks!

File: web-console/src/bootstrap/react-table-custom-pagination.tsx [hunk 1]
```diff
@@ -123,7 +123,7 @@ export class ReactTableCustomPagination extends React.PureComponent<
                   value={tempPage || String(page + 1)}
                   onBlur={this.applyTempPage}
                   onKeyPress={e => {
-                    if (e.which === 13 || e.keyCode === 13) {
+                    if (e.key === 'enter') {
```
Comment (vogievetsky) [inline web-console/src/bootstrap/react-table-custom-pagination.tsx:126]: I think the correct string here is `'Enter'` not `'enter'`

File: web-console/src/views/query-view/query-output/column-rename-input/column-rename-input.tsx [hunk 2]
```diff
@@ -44,12 +44,12 @@ export const ColumnRenameInput = React.memo(function ColumnRenameInput(
       value={newName}
       onChange={(e: any) => setNewName(e.target.value)}
       onKeyDown={(e: React.KeyboardEvent<HTMLInputElement>) => {
-        switch (e.keyCode) {
-          case 13: // Enter
+        switch (e.key) {
+          case 'enter': // Enter
```
Comment (vogievetsky) [inline web-console/src/views/query-view/query-output/column-rename-input/column-rename-input.tsx:48]: Ditto on the capitalization here and everywhere

Comment (vogievetsky) [reply web-console/src/views/query-view/query-output/column-rename-input/column-rename-input.tsx:48]: Also now we do not need the comment. Yay for self documenting code.

**Review by vogievetsky — CHANGES_REQUESTED** [review]
This is a great change, but can you please check the correct capitalization? Does this work?

File: web-console/src/views/query-view/query-output/column-rename-input/column-rename-input.tsx (hunk 2 above)
Comment (AshishKapoor) [reply web-console/src/views/query-view/query-output/column-rename-input/column-rename-input.tsx]: Done! Changed to 'Enter' and 'Tab'. Also, removed unwanted comments.

For testing and reference https://css-tricks.com/snippets/javascript/javascript-keycodes/

File: web-console/src/bootstrap/react-table-custom-pagination.tsx (hunk 1 above)
Comment (AshishKapoor) [reply web-console/src/bootstrap/react-table-custom-pagination.tsx]: Done!

**Comment by AshishKapoor** [comment]
> This is a great change, but can you please check the correct capitalization? Does this work?

Can you suggest steps to test these changes?

**Review by vogievetsky — APPROVED** [review]
Great stuff!

**Pull request merged**